docs/technical_reference/unit_models/treatment_models/med_md_semibatch.rst

```diff
@@ -0,0 +1,156 @@
+
+:math:`dt`: If :math:`None` is provided to time interval, the model uses default equations to determine its value based on the specified feed flow rate to the VAGMD module.
+
+:math:`Cap_{med}`: MED capacity should be associated with a single MD module for the batch process simulation, before scaled to a certain capacity.
+This value should be selected so that there's enough MED brine can be fed to the MD component.
+
+:math:`T_{cooling,in}`: The cooling water inlet temperature is not required when cooling system type is set to "closed". See details in the VAGMD base model documentation.
+
+:math:`md\_high\_brine\_salinity`: Specify/predict if the MD component will operate at high brine salinity mode (>175.3 g/L)
+
+Variables connected beteen periods
+----------------------------------
+This table lists paris of variables that need to be connected across two time periods during processing phase
+
+.. csv-table::
+   :header: "Variable from the previous period", "Symbol", "Variable in the next period", "Symbol"
+
+   "Temperature from S2", ":math:`T_{S2}`", "Feed temperature to MD", ":math:`T_{in,MD}`"
+   "TDS mass flow rate from S2 to MD", ":math:`m_{TDS,to-MD}`", "TDS mass flow rate fed to MD", ":math:`m_{TDS,MD}`"
+   "H2O mass flow rate from S2 to MD", ":math:`m_{H2O,to-MD}`", "H2O mass flow rate fed to MD", ":math:`m_{H2O,MD}`"
+   "Flow pressure from S2", ":math:`P_{S2}`", "Feed pressure to VAGMD", ":math:`P_{in,MD}`"
+   "Temperature from S2", ":math:`T_{S2}`", "Temperature of liquid remained in M1", ":math:`T_{in,MD}`"
+   "TDS mass flow rate from S2 and remained in M1", ":math:`m_{S2,remained}`", "TDS mass flow rate remained in M1", ":math:`m_{TDS,remained}`"
+   "H2O mass flow rate from S2 and remained in M1", ":math:`m_{S2,remained}`", "H2O mass flow rate remained in M1", ":math:`m_{H2O,remained}`"
+   "Flow pressure from S2", ":math:`P_{S2}`", "Pressure of liquid remained in M1", ":math:`P_{remained}`"
+   "Liquid volume in tank", ":math:`V_{tank}`", "Tank volume from previous step", ":math:`V_{tank,pre}`"
+
+These variables will be unfixed in each period after connected to the previous step to maintain the degree of freedom.
+
+Model details
+-------------
+
+* Unit model integrated in the flowsheet
+  
+For the technical details of LT-MED and VAGMD model, please refer to their documentation in the REFLO package.
+
+For the Mixer and Separator unit model, please refer to the documentation of IDAES package.
+
+* Initial condition at the first period
+
+VAGMD feed temperature and concentration are the same as MED brine at the beginning of the processing progress, 
+and so is the mixed liquid in M1, assuming the initial volume in M1 is the user specified batch volume.
+The MED brine tank starts as empty.
+
+* Discharging and refilling phase 
+
+The length of discharging and refilling phase is determined by the following equation:
+
+:math:`T_{reflling} = \frac{V_{batch} - V_{tank_stored}}{\nu_{MED,brine} - \nu_{MD,distillate}}`
```

Comment:
```suggestion
:math:`t_{refill} = \frac{V_{batch} - V_{tank,stored}}{\nu_{MED,brine} - \nu_{MD,distillate}}`
```
This relationship isn't clear to me. For one, during refill/discharge mode, what is the MD distillate flow rate? I could understand the MED continuing operation. Is the MED brine flow rate assumed constant through this whole period? I figured MED brine flow rate can vary with solar thermal input but perhaps the temporal resolution of the refill phase is high enough to ignore such variability.

Reply:
The MD is operating during the refill/discharge phase, with a direct feed from MED brine as assumed. 

And yes, the MED brine flow rate could vary if the refilling/discharge phase is very long, because of a large batch volume selected by the user. In normal cases where the refilling phase takes only a couple of minutes, the resolution should be enough. We can always come back and make this phase a multiperiod structure too if needed in some cases.